### PR TITLE
tested LXD with MAAS 2.3. Small updates and details surrounding LXD c…

### PR DESCRIPTION
**Pull request by degville**
- tested LXD with MAAS 2.3
- small updates and details surrounding LXD configuration with MAAS
- renamed bridge to be the same as the lxd init default

### CODE REVIEW

File: en/installconfig-lxd-install.md [hunk 1]
```diff
@@ -4,16 +4,25 @@ TODO:  Text needs a review
 
 # Install with LXD
 
+[LXD][link-lxd] is a next generation system container manager for Linux
```
Comment (pmatulis) [inline en/installconfig-lxd-install.md]: I know it's what upstream says but it sounds both extravagant and vague to me.

File: en/installconfig-lxd-install.md [hunk 2]
```diff
@@ -4,16 +4,25 @@ TODO:  Text needs a review
...
-- a network bridge on the LXD host (e.g. br0)
+- a network bridge on the LXD host (e.g. lxdbr0)
 - LXD and ZFS
 - a container profile
 
+!!! Note:
+    The LXD project was founded and is currently led by
+    [Canonical Ltd][link-canonical] with contributions from a range of other companies 
+    and individual contributors.
+
```
Comment (pmatulis) [inline en/installconfig-lxd-install.md]: I don't see why we need to include this in the MAAS docs.

File: en/installconfig-lxd-install.md (hunk 1 above)
Comment (degville) [reply en/installconfig-lxd-install.md]: I guess I put it there more for consistency. I did make an attempt to tone it down though.

File: en/installconfig-lxd-install.md (hunk 2 above)
Comment (degville) [reply en/installconfig-lxd-install.md]: Just to give some context. This and the para above is really if anyone is browsing the install docs to see what's possible and may not be familiar with LXD.

**Comment by degville** [comment]
Thanks for the feedback @pmatulis. Having given it some thought, I think you're right. I've toned down the intro slightly and removed the note.

**Pull request merged**